Sources/PointFree/Database.swift

```
@@ -159,6 +184,29 @@ public struct Database {
   }
 }
 
+private func createFeedRequestEvent(
+  type: Database.FeedRequestEvent.FeedType,
+  userAgent: String,
+  userId: Database.User.Id
+  ) -> EitherIO<Error, Prelude.Unit> {
+
+  return execute(
+    """
+    INSERT INTO "feed_request_events"
+    ("type", "user_agent", "user_id")
+    VALUES
+    ($1, $2, $3)
+    ON CONFLICT DO NOTHING
```

doing this so that we just silently ignore any user agents we've already seen